docs/conf.py

```diff
@@ -17,6 +17,9 @@
     "sphinx_copybutton",
 ]
 html_theme = "sphinx_book_theme"
+html_theme_options = {
+    "collapse_navigation": False,
```

Comment:
ChatGPT. There are no such `html_theme_options` in [`sphinx-book-them`](https://sphinx-book-theme.readthedocs.io/en/stable/search.html?q=html_theme_options) or [`pydata-sphinx-theme`](https://pydata-sphinx-theme.readthedocs.io/en/stable/search.html?q=html_theme_options) from which it inherits. What you probably want is [this](https://sphinx-book-theme.readthedocs.io/en/stable/sections/sidebar-primary.html).